nncf/experimental/openvino_native/quantization/algorithms/min_max/openvino_backend.py

```diff
@@ -0,0 +1,130 @@
+"""
```

Comment:
We should place algorithm backends near the algorithm.

Reply:
We decided to put all files specific to the OpenVINO Native backend in `experimental` dir.